src/pages/en/core-concepts/component-hydration.md

```diff
@@ -90,6 +90,8 @@ Start importing the component JS at page load and hydrate when the import comple
 
 If more than one renderer is included in the Astro [config](/en/reference/configuration-reference), `client:only` needs a hint to know which renderer to use for the component. For example, `client:only="react"` would make sure that the component is hydrated in the browser with the React renderer. For custom renderers not provided by `@astrojs`, use the full name of the renderer provided in your Astro config, i.e. `<client:only="my-custom-renderer" />`.
 
+📚 See our [directives reference](/en/reference/directives-reference#ui-framework-components) page for more infomation on all of the `client:` directives.
```

Comment:
This page can probably be removed entirely now. It was split into "framework-components" and "partial-hydration." When doing so, we should also make a Netlify redirect to partial-hydration. (I think anyone linking to Astro's hydration previously was probably referring more to the concept, than the hydration of the framework components themselves, so that's my recommendation.)

I'll make a separate PR for this.  :)